Dockerfile-linux.template

```
@@ -329,7 +335,9 @@ RUN set -eux; \
 		--with-sqlite3=/usr \
 		\
 		--with-curl \
-		--with-iconv=shared \
+{{ if is_alpine then ( -}}
+		--with-iconv=/usr \
+{{ )  else "" end -}}
```

I think it wouldn't hurt to always be explicit here (as we've done with several of the others in this section):

```suggestion
		--with-iconv{{ if is_alpine then "=/usr" else "" end }} \
```